packages/http-client/src/http-client.ts

```diff
@@ -301,18 +332,62 @@ function logRequestInformation(request: HttpRequestConfig) {
   }
 }
 
+/**
+ * Builds a [[DestinationHttpRequestConfig]] for the given destination, merges it into the given `requestConfig`
+ * and executes it (using Axios).
+ * The overload, that accepts [[HttpRequestConfigWithOrigin]] as a parameter, is deprecated and replaced the function [[executeHttpRequestWithOrigin]].
+ * @param destination - A destination or a destination name and a JWT.
+ * @param requestConfig - Any object representing an HTTP request.
+ * @param options - An [[HttpRequestOptions]] of the HTTP request for configuring e.g., CSRF token delegation. By default, the SDK will fetch the CSRF token.
+ * @returns A promise resolving to an [[HttpResponse]].
+ */
+export function executeHttpRequest<T extends HttpRequestConfig>(
+  destination: DestinationOrFetchOptions,
+  requestConfig: T,
+  options?: HttpRequestOptions
+): Promise<HttpResponse>;
+/**
+ * @deprecated This overload is replaced by the function [[executeHttpRequestWithOrigin]].
+ */
+export function executeHttpRequest<T extends HttpRequestConfigWithOrigin>(
+  destination: DestinationOrFetchOptions,
+  requestConfig: T,
+  options?: HttpRequestOptions
+): Promise<HttpResponse>;
+// eslint-disable-next-line jsdoc/require-jsdoc
+export function executeHttpRequest<
+  T extends HttpRequestConfig | HttpRequestConfigWithOrigin
+>(
+  destination: DestinationOrFetchOptions,
+  requestConfig: T,
+  options?: HttpRequestOptions
+): Promise<HttpResponse> {
+  // eslint-disable-next-line jsdoc/require-jsdoc
```

Comment:
[req] This comment seems like a mistake?

Reply:
> 365:3  error  Missing JSDoc comment  jsdoc/require-jsdoc

🤷

Reply:
I'll create an issue for them